### PR TITLE
Drawer set up

### PR DESCRIPTION
**Pull request by arwadley**
1. Rendering static html from server. 
2. basic drawer set-up complete. 
3. However effects and + rotating to x on-click. 
4. I have a lot of repeated code for all of the different component classes; I am considering re-factoring to a parent/child class system to cut that out. However, I am wary of doing that as the on-click that changes the pluses also needs to display the informational views specific to each page. Still considering the best way to deal with that.

### CODE REVIEW

**Review by mfuechec — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "scripts": {
+    "test": "jest",
+    "start": "webpack -d --watch & live-server --port=3030 --watch=dist --open=dist",
+    "start-oh": "webpack -d --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": ""
+  },
+  "author": "",
```
Comment (mfuechec) [inline package.json]: Quit being humble, you are the author.

File: client/index.jsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import ReactDOM from 'react-dom'; 
+import './App.jsx'
+
+ReactDOM.render(<App/>, document.getElementById('#root'));
```
Comment (mfuechec) [inline client/index.jsx]: It works for me with just 'root'. Surprised it works this way too.

File: client/Components/AssemblyAndDocuments.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'; 
+
+export default class AssemblyAndDocuments extends React.Component{
```
Comment (mfuechec) [inline client/Components/AssemblyAndDocuments.jsx:3]: I'm surprised everything is in a class component, but I don't know of any actual reasons why they shouldn't be, so carry on.

**Review by mfuechec — APPROVED** [review]
I made some style suggestions. Most of them seemed to already be handled in a later commit though.

**Comment by arwadley** [comment]
They're all class components so they can hold the state that controls when
the + turns to x and the drawer pulls down on-click. What you're seeing
there just doesn't have the functionality for the drawer pull down built
out yet.

**Pull request merged**